src/CustomersList/Header/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './style.css';
+
+function  Header(){
+    return(
+          <>
```

Comment:
Again, react fragments are not required since first div is encompassing all its children